class_defines.py

```diff
@@ -1,3 +1,4 @@
+from distutils import core
```

Comment:
This looks like a leftover import

Reply:
good observations, thanks for the review.   PR updated to fix.